docs/release_notes/release_notes.md

```diff
@@ -8,111 +8,45 @@ has_children: true
-            * Authentication - Excessive Failed VPN Logins for a User
-            * Authentication - Excessive Failed VPN Logins from a Source
-
-    * Better support for DNS logs
-        * Added props and field extractions for Windows DNS Logs (MSAD:NT6:DNS, isc:bind:query, & isc:bind:queryerror sourcetypes).
+## Version 2.1.0 (May 2022)
 
 
+* ### Ransomware alert improvement
+    * Ignored below file paths in the `Ransomware - Calculate UpperBound for Spike in File Writes` and `Ransomware - Spike in File Writes` alerts to reduce the false positive.
+        * `C:\Program Files\*`
+        * `C:\Program Files (x86)\*`
+        * `C:\Windows\Temp\*`
+        * `C:\ProgramData\Tenable\Nessus\nessus\plugins\*`
+        * `C:\Windows\ServiceProfiles\LocalService\AppData\Local\Temp\*`
+        * `C:\Users\*\AppData\Local\Temp\*`
 
-* ### Bug Fixes and Typos
-    * Renamed the `0365 - O365 Service is not Operational` alert to `O365 - O365 Service is not Operational` since there was an accidental typo present.
-        * Users will be required to reconfigure the new alert.
-    
-    * Improvements to Office 365 Active Directory related alerts and dashboards. It now displays some of the missing events.
+    * Increased minimun file write limit from 20 to 3000 to reduce the false positive.
+
+    * Added `top5_file_extension`, `avg`, and `stdev` field in the `Ransomware - Spike in File Writes` alert.
+
+    * Added `parent_process_path` field in the `Ransomware - Endpoint Compromise - Fake Windows Processes` alert.
 
-    * Fixed some issues with Python commands.
-        * Fixed an issue when handling an empty ip field for the device_inventory_gen command.
-        * Fixed a time-out issue with Malicious IP upload and download commands.
+* ### Palo Alto Firewall System alert and dashboard improvement
+    * Excluded the License related events from the `Palo Alto High System Alert` and `System Events` panel. 
+    * Addded new `License Events` panel to show the all license events.
 
-    * Renamed the some of the panel titles in the `DNS Tracker`, `GSuite` `Kaspersky`, `Office 365` and `Windows Defender` dashboard.
+* ### Palo Alto Firewall DNS Sinkhole improvement
+    * Added `url` field in the `Palo Alto DNS Sinkhole` alert and forensic searches.
+
+* ### DNS Tracker dashboard improvement.
+    * Added hostname detail in the Requester field to easily identify the machine.
+    * Added new panels to show information on DNS to DNS traffic.
+
+* ### Bug Fixes and Typos
+    * Fixed a drilldown issue for the `Antivirus` panel in the `Overview` dashboard.
 
-    * Fixed a drilldown issue for the Antivirus dashboard panel in the Overview dashboard.
+    * Fixed the token name to populate the `PowerShell Script Execution Error` panel of `Microsoft 365 Defender ATP Audit` dashboard.
 
+    * Fixed the `Linux/Unix` dashboard to show results with null field values when "All" is selected in the filter. 
```

Comment:
* Fixed the `Linux/Unix` dashboard to show results when some fields in the data are not present.